clang/lib/Driver/ToolChains/Cuda.cpp

```diff
@@ -61,7 +61,7 @@ static CudaVersion ParseCudaVersionFile(llvm::StringRef V) {
     return CudaVersion::CUDA_92;
   if (Major == 10 && Minor == 0)
     return CudaVersion::CUDA_100;
-  if (Major == 10 && Minor == 1)
+  if (Major == 10 && (Minor == 1 || Minor == 2))
```

Comment:
ToT Clang lags behind on released CUDA versions.

Reply:
We are hoping upstream clang catches up with 10.2 support, this change allows to use it on the meantime. Their release notes and our testing shows 10.2 can work as if 10.1 and there are no regressions. Otherwise, the default installation of many people simply fails to compile any SYCL for CUDA application.

```suggestion
// Enable CUDA 10.2 toolkit acting as 10.1 until tip clang catches up
  if (Major == 10 && (Minor == 1 || Minor == 2))
```

Reply:
Actually, upstream llvm patch https://github.com/llvm/llvm-project/commit/12fefeef203ab4ef52d19bcdbd4180608a4deae1  (https://reviews.llvm.org/D73231)   is a better fix, so we won't need once this lands into the intel/llvm fork